bf2fc6cc711aee1a0c2a/kas-fleet-manager
# fix: Fixed required list has not defined properties:['items'] bug

## Description
This issue was picked up by the monthly continuous API scanning run by Product Security.

The RapidPT project is now using the validator https://github.com/p1c2u/openapi-spec-validator to validate OpenAPI definition files. It is recommended that the OpenAPI definitions have been validated to be scanned successfully. kafkas_mgmt definition failed to be validated with the following error, although it didn't result in a parsing error by the scanning tool so the API was successfully scanned)

`$ openapi-spec-validator 202110-RHOSAK/apis/kafkas_mgmt_api_definition.json `
   `Required list has not defined properties: ['items']`

## Verification Steps
1. Go to line 832 the changes were made here.
2. Make sure to have OpenAPI spec validator installed. Can be found [here](https://github.com/p1c2u/openapi-spec-validator)
3. Open a terminal in the openapi folder. 
4. Run the command `openapi-spec-validator kas-fleet-manager.yaml` 
5. It should respond with Ok.

## Checklist (Definition of Done)
